model/vm_host.rb

```diff
@@ -353,16 +353,25 @@ def init_health_monitor_session
     }
   end
 
-  def disk_devices
-    storage_devices.each { |sd| sd.set_underlying_unix_devices if sd.unix_device_list.nil? || sd.unix_device_list.empty? }
+  def disk_device_ids
+    # we use this next line to migrate data from the old formatting (storing device names) to the new (storing id) so we trigger the convert
+    # whenever an element inside unix_device_list is not a SSD or NVMe id.
+    # SSD and NVMe ids start with wwn or nvme-eui respectively.
+    # This next line can be removed in the future after the first run of the code.
```

Comment:
Maybe add a YYY prefix so we can grep this as a thing to do. We have used this prefix in few other places for the same purpose.

```suggestion
    # YYY: This next line can be removed in the future after the first run of the code.
```

Reply:
done thanks.